### DISCUSSION TITLE
Shouldn't there be a new paragraph marker after a section marker?

### DISCUSSION THREAD

**Discussion by kavitharaju** [usfm-bible/tcdocs#25] [General]
The following test cases in this repo have positive test cases that do not contain a new paragraph marker(`\p`, `\q` or any such markers)  before the next verse(`\v` and versetext) after a section marker(`\s#`).
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/specExamples/character/origin.usfm#L21
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/specExamples/titles/origin.usfm#L15

In the following case there is no paragraph start after an `\esb`-`\esbe`.
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/esb/origin.usfm#L17

Also the use of `\s5` marker in several of usfmjsTests have verse contents occurring without a paragraph start after it.
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/isa_verse_span/origin.usfm#L13
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/isa_footnote/origin.usfm#L12
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/tit_extra_space_after_chapter/origin.usfm#L17
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/1ch_verse_span/origin.usfm#L11
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/usfmBodyTestD/origin.usfm#L21
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/acts_1_milestone.oldformat/origin.usfm#L9
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/nb/origin.usfm#L4
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/usfmIntroTest/origin.usfm#L64
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/usfm-body-testF/origin.usfm#L27
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/out_of_sequence_verses/origin.usfm#L26
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/acts_1_milestone/origin.usfm#L11
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/luk_quotes/origin.usfm#L22
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/samples-from-wild/doo43-1/origin.usfm#L13
- https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/samples-from-wild/doo43-2/origin.usfm#L13

**Comment by KentSpiel** [comment]
- I agree that a paragraph marker is required after a section marker  before a verse marker.
- I do not think one is required after  \esb . . \esbe if there is one before the sidebar. In other words ([tcdocs/tests/usfmjsTests/esb/origin.usfm](https://github.com/usfm-bible/tcdocs/blob/471a15103622813ddd4e4eb13b17bb464bf02cbf/tests/usfmjsTests/esb/origin.usfm#L17))  is OK.
- The \s5 without a paragraph marker before the verse is not OK. It is the same as the first two examples.
